x11-terms/terminator/terminator-1.92.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_6 python3_7 )
+inherit eutils distutils-r1 virtualx xdg
```

Comment:
I don't see eutils anywhere in use here. xdg is overriding phase functions, you should make sure that it does not mess with distutils-r1 which does the same (src_prepare in particular). Order is important here. You can avoid this by using xdg-utils instead.

Reply:
I moved xdg to the same position where gnome2 was in the previous ebuild. xdg-utils doesn't provide postinst used here.

Reply:
But xdg-utils provides the functions that are run inside pkg_postinst.

Reply:
Okay